Sources/Impl/PinLayoutImpl+Warning.swift

```diff
@@ -152,7 +152,7 @@ extension PinLayoutImpl {
             #endif
         }
 
-        displayText += ", Tag: \(view.tag))\n"
+        displayText += ", Debug description: \(view.debugDescription))\n"
```

Comment:
In fact @antoinelamy, I would have put everything in the debugDescription property, anyway we will need to do that for other Layoutable.

I would include all this: 
```
    var hierarchy: [String] = []
    ...
     if hierarchy.count > 0 {	  
             
     }
```

Reply:
@lucdion not sure about that one, `debugDescription` is already implemented on NSView/UIView and it's already pretty verbose. Furthermore, we cannot override an existing function from an extension so I would leave the responsibility to the caller with special debugging needs to override `debugDescription` in their subclasses. What do you think @lucdion ?